### PR TITLE
Mimir 1490 tab navigation improvments highcharts

### PR DESCRIPTION
**Pull request by Carl-OW**

### CODE REVIEW

File: src/main/resources/assets/styles/main.scss [hunk 1]
```diff
@@ -136,3 +136,8 @@ section.part-divider + section.part-divider {
     width: 95%;
   }
 }
+
+// Highcharts Keyboard navigation tab fix (accessibility elements fix)
+.highcharts-a11y-proxy-button.highcharts-no-tooltip{
```
Comment (ssb-cgn) [inline src/main/resources/assets/styles/main.scss:141]: Try to move this to _highchart.scss instead :)

**Review by ssb-cgn — CHANGES_REQUESTED** [review]
Se comment, move change to highchart.scss

File: src/main/resources/assets/styles/main.scss (hunk 1 above)
Comment (Carl-OW) [reply src/main/resources/assets/styles/main.scss:141]: ok

**Comment by Carl-OW** [comment]
Highcharts Keyboard navigation tab fix (accessibility elements fix)

**Review by ssb-cgn — APPROVED** [review]
Nice 🥇

**Pull request merged**